packages/origin-247-transfer/package.json

```diff
@@ -0,0 +1,71 @@
+{
+    "name": "@energyweb/origin-247-transfer",
+    "version": "0.0.1",
+    "main": "./dist/js/src/index.js",
+    "types": "./dist/js/src/index.d.ts",
+    "scripts": {
+        "test:unit": "jest --testRegex '\\.spec\\.ts$'",
+        "test:e2e": "concurrently --success first --kill-others -n eth,test \"yarn run:ganache\" \"wait-on tcp:8546 && yarn drop && yarn migrate && yarn test:e2e:jest\"",
+        "test:e2e:jest": "WEB3=http://localhost:8546 jest --testRegex '.e2e-spec.ts$' --forceExit",
+        "build": "tsc -b tsconfig.json",
+        "migrate": "yarn typeorm:run && yarn typeorm:run:issuer",
+        "drop": "yarn typeorm:drop && yarn typeorm:drop:issuer",
+        "typeorm": "ts-node -r tsconfig-paths/register node_modules/typeorm/cli.js --config ormconfig-dev.ts",
+        "typeorm:drop": "yarn typeorm schema:drop",
+        "typeorm:run": "yarn typeorm migration:run",
+        "typeorm:run:issuer": "node_modules/typeorm/cli.js migration:run --config node_modules/@energyweb/issuer-api/dist/js/ormconfig.js",
+        "typeorm:drop:issuer": "node_modules/typeorm/cli.js schema:drop --config node_modules/@energyweb/issuer-api/dist/js/ormconfig.js",
+        "run:ganache": "ganache-cli -q -m 'govern long helmet alert stay supply kick knife boss until buzz unlock' -l 8000000 -e 1000000 -a 20 -p 8546",
+        "publish:canary": "lerna publish --yes --skip-git --exact --cd-version=prerelease --pre-dist-tag canary --preid=alpha.$BUILD_ID",
+        "publish:release": "lerna version --create-release github --conventional-commits --exact --yes --message \"chore(release): publish /skip-deploy\" && lerna publish from-git --yes"
+    },
+    "dependencies": {
+        "@nestjs/common": "7.6.18",
+        "@nestjs/core": "7.6.18",
+        "@nestjs/cqrs": "7.0.1",
+        "@nestjs/typeorm": "7.1.5",
+        "ethers": "5.4.1",
+        "pg": "8.6.0",
+        "typeorm": "0.2.34",
+        "lodash": "4.17.21"
+    },
+    "peerDependencies": {
+        "@energyweb/origin-247-certificate": "^0.0.2-alpha.1626083770.0"
```

Comment:
You are still referencing the NPM package here? Can we also make this one use the local version?

Reply:
This is `peerDependency` so it means, that anybody using this package have to install this module by himself, because this package doesn't contain it.

Reply:
I'm not sure if we have setup standards in Origin for such dependencies (like issuer-api depends on issuer for example)

Reply:
Sure, I just wonder how the release process will work with Lerna. Hope that Lerna will detect the dependency and auto-increment this version to the latest released version during the release.

Reply:
We have to monitor this 247-sdk because probably there will be some release/version problems that will have to be fixed. It's very probable that I missed something.